conans/test/integration/install_missing_dep_test.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from conans.test.utils.tools import TestClient, TestServer
+from conans.model.ref import ConanFileReference, PackageReference
+import os
+from conans.test.utils.cpp_test_files import cpp_hello_conan_files
+from conans.util.files import load, save
+from time import sleep
+import time
+from conans.paths import CONAN_MANIFEST
+
+
+class InstallMissingDependency(unittest.TestCase):
+
+    def missing_dep_test(self):
+        test_server = TestServer()
+        self.servers = {"myremote": test_server}
+        self.client = TestClient(servers=self.servers, users={"myremote": [("lasote", "mypass")]})
+
+        # Create deps packages
+        dep1_conanfile = """from conans import ConanFile
+class Dep1Pkg(ConanFile):
+    name = "dep1"
```

Comment:
Name also unnecessary.

Reply:
I know, but I prefer to keep the name in the recipe... at least to know but that `conanfile.py` is about. Is there any reason (internal to the machinery of Conan) to leave it out?

Reply:
Fair enough, you can leave the name.